pkg/await/awaiters.go

```diff
@@ -55,6 +56,12 @@ func (cac *createAwaitConfig) eventClient() (dynamic.ResourceInterface, error) {
 	}, cac.currentInputs.GetNamespace())
 }
 
+func (cac *createAwaitConfig) log(sev diag.Severity, message string, isStatus bool) {
```

Comment:
what about splitting this into `log` and `logStatus`, like in the pulumi/pulumi PR?